docs/build-with-powerloom/snapshotter-node/lite-node/Telegram Bot Setup.md

```diff
@@ -0,0 +1,48 @@
+### Setting up your Telegram Bot | Node Health Monitoring 
+
+
+This guide provides instructions for setting up your Powerloom Telegram Bot to monitor the health of your node. The Snapshotter-lite nodes currently require manual monitoring to detect any issues.
+
+Introducing the Powerloom Telegram Bot, ‘PowerloomReportingBot’. This bot service maintains an internal health state and reports any issues to operators, such as failed or missed snapshots. 
+
+### Follow the steps below to set up your Node Health Monitoring process using Telegram.
+
+- Step 1: Find the telegram bot by searching ‘PowerloomReportingBot’ on Telegram
+- Step 2: Type ‘/start’ to activate the bot. This will provide you with a Chat ID.
+
+![chatID-start](/images/chatID-start.png)
+
+### If you are a new Snapshotter,
+Enter the ChatID when prompted on the terminal during the node setup. 
+
+### If you are an existing Snapshotter, 
```

Comment:
It's not about being a 'new snapshotter' or 'existing snapshotter'. It is about whether they have a previously existing installation.